macros/core/PGbasicmacros.pl

```diff
@@ -3023,7 +3040,14 @@ sub image {
 	while (@image_list) {
 		my $image_item = shift @image_list;
 		if (ref $image_item eq 'parser::GraphTool') {
-			push(@output_list, $image_item->generateAnswerGraph(ariaDescription => shift @alt_list // ''));
+			push(
+				@output_list,
+				$image_item->generateAnswerGraph(
+					width           => $width,
+					height          => $height,
+					ariaDescription => shift @alt_list // ''
```

Comment:
I said earlier a sane default for the graph tool was a width of 200, but what I really should have said is a sane minimum for the width is 200.  Really, that is generally still to small.  The sane default is really what the graph tool uses by default to begin with.  That is why it uses that.  That also has the added advantage of being a responsive size.  On narrow screens it is smaller than on wide screens.  So it would be better here if the width or height were only given in the case that one of them is explicitly given.  Then if neither are given, the graph tool will use the true sane defaults.